### PR TITLE
Add types for admin-rebalance subcommands

### PR DESCRIPTION
**Pull request by krisis**
.. and for tracing rebalance internal functions.

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

File: rebalance.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+		PercentFreeGoal: rm.PercentFreeGoal,
+		PoolStats:       rm.PoolStats,
+		StoppedAt:       rm.StoppedAt,
+	}
+	return json.Marshal(r)
+}
+
+func (rm *RebalanceMeta) UnmarshalJSON(b []byte) error {
+	type rebalMeta struct {
+		ARN             uuid.UUID
```
Comment (vadmeste) [inline rebalance.go:85]: This doesn't have json tags like MarshalJSON(), it doesn't seem this will work correctly

**Review by vadmeste — COMMENTED** [review]
One comment

File: rebalance.go [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+	if err != nil {
+		return arn, err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return arn, httpRespToErrorResponse(resp)
+	}
+
+	var rebalInfo struct {
+		ARN uuid.UUID `json:"arn"`
```
Comment (harshavardhana) [inline rebalance.go:115]: ARN has a specific meaning in AWS S3 spec not sure it would be wise to use UUID as ARN - you can just call it as ID @krisis

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
Please rebase the PR @krisis

**Comment by harshavardhana** [comment]
v1.6.1 is pushed @krisis FYI.

**Pull request merged**